src/tools/rust-analyzer/crates/hir-ty/src/layout/tests.rs

```diff
@@ -150,7 +150,7 @@ fn check_size_and_align(
 ) {
     let l = eval_goal(ra_fixture, minicore).unwrap();
     assert_eq!(l.size.bytes(), size, "size mismatch");
-    assert_eq!(l.align.abi.bytes(), align, "align mismatch");
```

Comment:
Remark: From a review perspective it's unclear whether these rust-analyzer changes are intended/accurate (and not just a find-replace mishap), but they seem plausible and I guess CI would complain if they were broken, so it seems fine to rubber-stamp them.